### PR TITLE
Pull Boilerplate Code & Update update.cfg

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

# TL;DR
Updated boilerplate in this repo. Added `flyte` entry in `update.cfg` file. I'll have to pull `flyte` repo from the `boilerplate` repo once this is merged.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by samhita-alla** [comment]
@kumare3 Can you let me know if this is the right way to update the boilerplate code? If so, we'll have to update the `update.cfg` file in the concerned repos which have the boilerplate repo, after which we'll have to update (`make update_boilerplate` command) the boilerplate code again to copy the [flyte](https://github.com/flyteorg/boilerplate/tree/master/boilerplate/flyte) repo which has the [yaml](https://github.com/flyteorg/boilerplate/blob/master/boilerplate/flyte/config.yml) file for the welcome bot.

Moreover, `boilerplate` isn't present in all `flyteorg` repos. Do we need to [add](https://github.com/flyteorg/boilerplate#adding-boilerplate-to-a-repository) it in all the repos?

**Comment by kumare3** [comment]
@samhita-alla yes you are right. This is how boilerplate works. And ideally it should be in all flyteorg repos

**Review by kumare3 — APPROVED** [review]

**Comment by kumare3** [comment]
cc @katrogan

File: boilerplate/update.sh [hunk 1]
```diff
@@ -33,13 +35,34 @@ if [ -z "$REPOSITORY" ]; then
...
+  # are not acceptable, unless `other_junk` is a comment
+  if [[ "$junk" != "" ]] && [[ "$junk" != '#'* ]]; then
+    echo "Invalid config! Only one directory is allowed per line. Found '$junk'"
+    exit 1
+  fi
+
+  dir_path="${OUT}/boilerplate/${directory}"
+  # Make sure the directory exists
+  if ! [[ -d "$dir_path" ]]; then
+    echo "Invalid boilerplate directory: '$directory'"
```
Comment (pmahindrakar-oss) [inline boilerplate/update.sh:58]: nit : with path $dir_path

**Review by pmahindrakar-oss — APPROVED** [review]

**Review by kumare3 — APPROVED** [review]
